### PR TITLE
Add color picker

### PR DESCRIPTION
**Pull request by kvinty**

### CODE REVIEW

File: grimpicker/LICENSE
Comment (OctopusET) [inline grimpicker/LICENSE:1]: I don't think it's needed, would you remove this?

Comment (kvinty) [reply grimpicker/LICENSE:1]: OK, sure

**Comment by OctopusET** [comment]
Sorry I made some mistake while I was squashing commits into one, can you repush it?

**Comment by kvinty** [comment]
Pushed it as a single commit (also added a line in README), you can not reopen this PR. Do you think this tool has a place here? Any modifications needed?

**Review by OctopusET — APPROVED** [review]
LGTM

**Pull request merged**